cpp/cmake/IceConfig.cmake

```diff
@@ -1,239 +1,16 @@
+# Copyright (c) ZeroC, Inc.
 
-if (NOT Ice_HOME)
-  if (DEFINED ENV{ICE_HOME})
-    set(Ice_HOME $ENV{ICE_HOME} CACHE PATH "Path to the Ice installation directory")
-  endif()
-endif()
+# get_filename_component(PACKAGE_PREFIX_DIR "${CMAKE_CURRENT_LIST_DIR}/../../../" ABSOLUTE)
 
-include(CMakeFindDependencyMacro)
-find_package(Threads REQUIRED)
-
-if(NOT DEFINED Ice_ARCHITECTURE)
-  if (CMAKE_LIBRARY_ARCHITECTURE)
-    set(Ice_ARCHITECTURE ${CMAKE_LIBRARY_ARCHITECTURE} CACHE STRING "Library architecture")
+if(NOT DEFINED PACKAGE_PREFIX_DIR)
```

Comment:
These lines are temporary and will be replace with the above line once we move into the Ice packaging.

This is the standard setup for cmake as `IceConfig.cmake` is placed at `<prefix>/lib/cmake/ice/IceConfig.cmake`